cfme/tests/generic_objects/test_definitions.py

```diff
@@ -14,11 +14,14 @@
 
 
 @pytest.mark.sauce
+@pytest.mark.tier(0)
 @pytest.mark.parametrize('context', [ViaREST, ViaUI])
 def test_generic_object_definition_crud(appliance, context, soft_assert):
     """
     Polarion:
-        assignee: dmisharo
+        assignee: jdupuy
+        casecomponent: GenericObjects
+        caseimportance: high
         initialEstimate: 1/30h
```

Comment:
Are you sure about it (2 min)?

Reply:
It is quick since it is just creating a generic object via Rest and the UI. Maybe ~5 min is closer, I can change it.